common/country_types/hfe_country_types_pirate.txt

```diff
@@ -1,12 +1,14 @@
 #
+
 hfe_pirate = {
 	has_difficulty_bonuses = yes
-	counts_for_victory = yes
+	counts_for_victory = no
 	pop_growth = yes
 	has_pulse_events = yes
 	randomize_ethos = yes
 	shuffle_pop_ethos = yes
 	needs_survey = no
+	has_complete_sensor = yes # THIS FIXES THE BUG
```

Comment:
what bug?

Reply:
The issue with pirates and surveillance, as well as pirates committing suicide by entering hostile systems. It also makes their navigation commands easier.